src/gui/ExportProjectDialog.cpp

```diff
@@ -194,9 +194,13 @@ ProjectRenderer::ExportFileFormats convertIndexToExportFileFormat(int index)
 	case 0:
 		return ProjectRenderer::WaveFile;
 	case 1:
+#ifdef LMMS_HAVE_OGGVORBIS
 		return ProjectRenderer::OggFile;
 	case 2:
+#endif
+#ifdef LMMS_HAVE_MP3LAME
 		return ProjectRenderer::MP3File;
+#endif // If neither ogg nor mp3 are available, index 1 is OOB; fall through to default case
```

Comment:
>  I don't think parsing strings is the best solution either.

The dialog mandates a file extension filter.  Parsing string is perfectly sane here.  I prefer enum approaches too and we need them elsewhere, but I see no logic flaw in `.ogg` = `OGG` at this point.  It's not going to change.